radeon-profile/ioctl_amdgpu.cpp

```diff
@@ -16,14 +16,16 @@ amdgpuIoctlHandler::amdgpuIoctlHandler(unsigned cardIndex) : ioctlHandler(cardIn
 bool amdgpuIoctlHandler::getSensorValue(void *data, unsigned dataSize, unsigned sensor) const {
 #if defined(DRM_IOCTL_AMDGPU_INFO) && defined(AMDGPU_INFO_SENSOR)
     struct drm_amdgpu_info buffer = {};
+    QString sensor_name; 
     buffer.query = AMDGPU_INFO_SENSOR;
     buffer.return_pointer = reinterpret_cast<uint64_t>(data);
     buffer.return_size = dataSize;
     buffer.sensor_info.type = sensor;
 
     bool success = !ioctl(fd, DRM_IOCTL_AMDGPU_INFO, &buffer);
     if (Q_UNLIKELY(!success)) {
-        qInfo("Requested sensor: %02X", sensor);
+        sensor_name = getSensorName(sensor);
+        qInfo("Requested sensor: %s", qUtf8Printable(sensor_name));
```

Comment:
`qInfo` will print in console on every refresh for release build so maybe `qDebug` would be better here?
```suggestion
        qDebug("Requested sensor: %s", qUtf8Printable(sensor_name));
```